src/partials/header.html

```diff
@@ -0,0 +1,84 @@
+<!-- Adaptive header, by Volodymyr Vladykin -->
+<header class="header">
+  <div class="header__container container container__content">
+    <!-- Logo, link to core page -->
+    <div class="logo">
+      <a class="logo__link" href="./index.html"
+        ><span class="logo__link--dark">english</span>excellence<span
+          class="logo__link--dark"
+          >.</span
+        ></a
+      >
+    </div>
+
+    <!-- Mobile menu button, will be hidden on desktop -->
+    <button class="header__mobile_menu_button" type="button">
+      <svg class="header__mobile_menu_icon" width="32" height="32">
+        <use href="./img/sprite.svg#icon-menu"></use>
+      </svg>
+    </button>
+
+    <!-- Main navigation -->
+    <nav class="nav">
+      <!-- Navigation menu (visible only on desktop) -->
+      <ul class="header__nav_list">
+        <li>
+          <a href="#" class="nav_link header__nav_link--dark">About us</a>
+        </li>
```

Comment:
formatted code please

Reply:
Not sure what wrong here with formatting, this is result of Prettier formatting

Reply:
Ok
We need to have call for check

Reply:
will fixed when we will check all project